browser/src/index.css

```diff
@@ -0,0 +1,13 @@
+:root {
+  --COLOR-BRAND-PRIMARY: #6941c6;
+  --COLOR-NAV-ICON: #d6bbfb;
+  --COLOR-NAV-ICON-SELECTED: #ffffff;
+  --COLOR-NAV-ICON-SELECTED-BG: #53389e;
+  --COLOR-AVATAR-FONT: #7f56d9;
+  --COLOR-AVATAR-BG: #f9f5ff;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Excellent 👏 Let's move the variables to a new file.

Reply:
Sure. What's convention in this case? A `Variables.css` file in the /src dir?

Reply:
I think I'd make a src/styles directory and then create a src/styles/common/variables.scss.